simple/rdm_tagged_peek.c

```diff
@@ -205,6 +205,12 @@ int main(int argc, char **argv)
 
 	ret = run();
 
+	if (ret == -ENODATA) {
+		hints->caps = FI_TAGGED;
+
+		ret = run();
+	}
```

Comment:
Let's just update the test to only ask for FI_TAGGED.  It doesn't use the message interfaces at all from what I can see.